pkgs/by-name/gi/git-blame-someone-else/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+}:
+
+stdenv.mkDerivation {
+  pname = "git-blame-someone-else";
+  version = "0";
+
+  src = fetchurl {
```

Comment:
Use `fetchFromGitHub`, lock the version using a commit ID.

Example: https://github.com/NixOS/nixpkgs/pull/348180/files